ui/components/ui/popover/popover.component.js

```diff
@@ -67,10 +69,11 @@ const Popover = ({
         >
           <h2 title="popover">
             {onBack ? (
-              <button
-                className="fas fa-chevron-left popover-header__button"
-                title={t('back')}
+              <ButtonIcon
+                iconName={ICON_NAMES.ARROW_LEFT}
+                ariaLabel={t('back')}
                 onClick={onBack}
+                color={Color.iconDefault}
```

Comment:
Adding size here to align with design system component but it feels very small cc @garrettbear @Akatori-Design. Request to add Size.MD to ButtonIcon

 
<img width="1440" alt="Screenshot 2023-02-08 at 6 49 12 PM" src="https://user-images.githubusercontent.com/8112138/217705176-b73b5b65-c3c5-468b-afa3-ce9153ae0d7e.png">